File: yggdrasil/examples/split_and_merge/src/split_and_merge_modelA.jl
using Printf

function modelA_function(in_val)
  out_val1 = 2 * in_val
  out_val2 = 3 * in_val
  @printf("modelA_function(%s) = (%s, %s)\n", in_val, out_val1, out_val2)
  return out_val1, out_val2
end

File: yggdrasil/examples/split_and_merge/src/split_and_merge_modelA.m
function [out_val1, out_val2] = split_and_merge_modelA(in_val)
  out_val1 = 2 * in_val;
  out_val2 = 3 * in_val;
  disp(sprintf('modelA_function(%f) = (%f, %f)', in_val, out_val1, out_val2));
end
